move_base/include/move_base/move_base.h

```diff
@@ -229,6 +229,12 @@ namespace move_base {
       move_base::MoveBaseConfig default_config_;
       bool setup_, p_freq_change_, c_freq_change_;
       bool new_global_plan_;
+
+      // for move_backwards_recovery
+      double move_backwards_distance_;
+      bool move_backwards_enabled_;
+      double move_backwards_velocity_;
```

Comment:
Also make sure that these are initialized in the same place (reconfigureCB).  conservative_reset_dist_ = config.conservative_reset_dist;